main/test/flix/Test.Exp.Fixpoint.Compose.flix

```diff
@@ -86,4 +86,14 @@ namespace Test/Exp/Fixpoint/Compose {
         let rb = solve c3 <+> c4;
         (ra, rb)
 
+    @test
+    def testFixpointCompose13(): Bool = {
+        let ans = query openRow(#{Coolio("quack!").}) select s from Duck(s);
+        ans == List#{"quack!"}
+    }
+
+    def openRow(p: #{Coolio(String)}): #{Coolio(String), Duck(String) | r} = {
```

Comment:
I would like some more examples: Say: 

- 1x with 1 predicate
- 2x with 2 predicates (just trying different orders)
- 4x with 3 predicates (just trying different orders)

Reply:
Something that checks the order is not important.

Reply:
I'm not sure I follow, you want this test copied with 2 and 3 predicates?